python/lsst/pex/config/configDictField.py

```diff
@@ -191,12 +200,15 @@ def __init__(
             raise ValueError(f"'itemtype' {_typeStr(itemtype)} is not a supported type")
         if dictCheck is not None and not hasattr(dictCheck, "__call__"):
```

Comment:
These errors could actually be consolidated into one, e.g.:
```
check_errors = []
for name, check in (("dictCheck", dictCheck), ("keyCheck", keyCheck), ("itemCheck", itemCheck)):
    if check is not None and not hasattr(check, "__call__"):
        check_errors.append(name)
if check_errors:
    raise ValueError(f"{', '.join(check_errors)} must be callable")
```

Reply:
Good idea, I just wanted to be minimally invasive in my approach but this sounds like an improvement.

Reply:
Done!